### DISCUSSION TITLE
[Enhancement]: Xbox

### DISCUSSION THREAD

**Discussion by CoadyKidd** [ciderapp/Cider#660] [Feature Request]
### The feature you would like

Is there any way you could add support to Xbox? I'd love to be able to stream apple music from my xbox

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by grolschie** [comment]
Excellent idea. The Xbox supports Dolby Atmos, which would be awesome if implemented.

**Comment by cryptofyre** [comment]
Won't be a very easy process at all. WebView 2 does not support DRM through anything other then HTTPS so there would have to be a ton of back-end changes which will result in much slower updates.

Not saying it won't happen, just **not currently**.

**Comment by grolschie** [comment]
Understood. Thanks for responding. All the best with the project.

**Comment by CoadyKidd** [comment]
is there anyway you'd be able to turn the app into a UWP app like how gamr13.github.io has with all the apps on there?

**Comment by quacksire** [comment]
One thing, DRM
